src/Autoloader.php

```diff
@@ -9,6 +9,8 @@
 
 defined( 'ABSPATH' ) || exit;
 
+require_once dirname( __DIR__ ) . '/src/Loop.php';
```

Comment:
I'm not sure why this is needed (without this the `Loop` class isn't recognized anywhere), perhaps the autoloader would need some adjustments?